projects/dsp-ui/src/lib/action/components/confirmation-dialog/confirmation-dialog.component.spec.ts

```diff
@@ -50,7 +60,8 @@ describe('ConfirmationDialogComponent', () => {
         TestBed.configureTestingModule({
             declarations: [
                 ConfirmationDialogComponent,
-                ConfirmationDialogTestHostComponent
+                ConfirmationDialogTestHostComponent,
+                ConfirmationMessageComponent
```

Comment:
I wonder if it would be better to mock `ConfirmationMessageComponent ` here since this is a unit test for the confirmation dialog.